app/src/main/java/com/example/fitjournal/MainActivity.kt

```diff
@@ -74,7 +74,9 @@ class MainActivity : ComponentActivity() {
                                     LibraryScreen(
                                         modifier = mainScreenModifier,
                                         libraryWorkoutState = libraryScreenViewModel.libraryWorkoutState
-                                    )
+                                    ) {
+                                        showChildFabs = it
```

Comment:
move function to parameter call

Reply:
Unfortunately that is a problem.
the Library screen has its own clickable modifier used to hide the keyboard when user clicks on the screen,  and that one kills the clickable modifier I have in the AppScreen, so the only way to achieve both behaviors for library is to do it this way.

Reply:
![image](https://github.com/Alex-A-Fit/FitJournal-Android/assets/64118964/bac4ced2-4e8e-4cb2-b43f-0047ea27bf79)

Reply:
this is what I mean. This does not break anything but makes it clear what that function is for